qp_shogun/shogun/__init__.py

```diff
@@ -26,7 +26,7 @@
     # aligner
     'Aligner tool': ['choice:[' +
                      '"utree", ' +
-                     # '"burst", ' +
+                     '"burst", ' +
```

Comment:
I would recommend putting BURST to the last, because the computational expense is UTree < Bowtie2 < BURST and the accuracy is opposite.

Reply:
This order is not represented anywhere as the GUI sorts the options alphabetically ...

Reply:
Got it!